src/inotify/version.cr

```diff
@@ -1,3 +1,3 @@
 module Inotify
-  VERSION = "0.1.0"
+  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
```

Comment:
You don't have to bother with this, I suggest to remove this file - does someone user it?

Reply:
I am just used to it being there and it doesn't hurt to have it.